tools/slang-unit-test/unit-test-com-host-callable.slang

```diff
@@ -19,14 +19,14 @@ public __extern_cpp int getGlobal()
     return intGlobal;
 }
 
-[COM]
+[COM("9E0FCAF0-DE40-4CF1-A6A4-FF24814D32F2")]
```

Comment:
Having this have the option to work work without requiring an explicit guid could be handy. Perhaps the guid is internally generated as a hash on the name.